resources/eventing/charts/nats/values.yaml

```
@@ -194,6 +194,20 @@ nats:
       annotations:
       # key: "value"
 
+# Authentication setup
+auth:
+  enabled: true
+  resolver:
+    ##############################
+    #                            #
+    # Memory resolver settings   #
+    #                            #
+    ##############################
+    type: memory
+    configMap:
```

```suggestion
    secret:
```
(nit) its a secret, not a configMap, let rename it for consistency?